R/pkg/R/functions.R

```diff
@@ -86,6 +86,22 @@ NULL
 #' df <- createDataFrame(data.frame(time = as.POSIXct(dts), y = y))}
 NULL
 
+#' String functions for Column operations
+#'
+#' String functions defined for \code{Column}.
+#'
+#' @param x Column to compute on. In \code{instr}, it is the substring to check. In \code{format_number},
```

Comment:
`instr` seems to be the odd one here. it's `instr(y, x)` - should there be doc on `y` first/too?